jvcl/run/JvMail.pas

```diff
@@ -402,6 +402,7 @@ procedure TJvMail.CreateMapiMessage;
       SetLength(FAttachArray, Attachment.Count);
       for I := 0 to Attachment.Count - 1 do
       begin
+        // Look for #0 splitting parameter in physical filename (lpszPathName) and virtual name (lpszFileName)
```

Comment:
Please remove `lpsz` prefix in `lpszPathName` and `lpszFileName` to the variable reference matches